bin/ch-pack-image

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+libexec=$(cd "$(dirname "$0")" && pwd)/../libexec/charliecloud
+. "${libexec}/base.sh"
```

Comment:
I believe the `$()` format is a bashism. This will need to be replaced with sh syntax if we plan to keep this script POSIX.

```